sdk/dotnet/Pulumi/Core/OutputUtilities.cs

```diff
@@ -61,7 +61,7 @@ public static async Task<bool> GetIsKnownAsync<T>(Output<T> output)
         /// </summary>
         /// <param name="output">The <see cref="Output{T}"/> to evaluate.</param>
         public static Task<T> GetValueAsync<T>(Output<T> output)
```

Comment:
Not changing the public exported method (yet). Though an argument can be made for removing this from the public API as it is dangerous.

Reply:
It was added not that long ago - mainly for unit tests I believe. It's in our "semi-public" OutputUtilities and has a comment "please don't use unless you know what you are doing".